app/src/main/res/values/strings.xml

```diff
@@ -240,4 +240,5 @@
   <string name="notifications_mention">%1$s mentioned you on %2$s.</string>
   <string name="toggle_view_button">Toggle view</string>
   <string name="about_rate_us"><u>Rate Us</u></string>
+  <string name="about_faq"><u>Frequently Asked Questions</u></string>
```

Comment:
<u> tag might have issues with translation. Please check #1258.

Reply:
I think underlines should be added afterwards by the app, they do not need to be translated, right?

Reply:
I found this method most suitable https://stackoverflow.com/questions/2394935/can-i-underline-text-in-an-android-layout .

Reply:
Should I change it using Java code?

Reply:
I got it.

Reply:
@ujjwalagrawal17 Can you briefly explain about not changing it in the java code instead?

Reply:
We can change it using java code also but I thought if we can do it in XML it would be simpler.